src/lwc/gsModal/gsModal.html

```diff
@@ -0,0 +1,31 @@
+<template>
+  <!-- modal start -->        
+    <template if:true={bShowModal}>
+       <section role="dialog" tabindex="-1" aria-labelledby="modal-heading-01" aria-modal="true" aria-describedby="modal-content-id-1" class="slds-modal slds-fade-in-open">
+          <div class="slds-modal__container">
+             <!-- modal header start -->
+             <header class="slds-modal__header">
+                <button class="slds-button slds-button_icon slds-modal__close slds-button_icon-inverse" title={butonLabel} onclick={closeModal}>
+                   <lightning-icon icon-name="utility:close"
+                      alternative-text="close"
```

Comment:
There should be an existing custom label you can use for this rather than to hardcode the alternate text. When a non-english user is using a screen reader, we want the page to read the button alternative text in right language.

Reply:
@force2b what do you think about borrowing PMM's reusable modal component? https://github.com/SalesforceFoundation/PMM/blob/master/force-app/main/default/lwc/modal/modal.html It's quite similar, and has a few extra features that might be useful to NPSP in the future.